sdk/core/azure-core-amqp/src/impl/rust_amqp/amqp/message_receiver.cpp

```diff
@@ -67,27 +119,76 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
       Azure::Core::_internal::AzureNoReturnPath(
           "MessageReceiverImpl is being destroyed while open.");
     }
-    if (m_link)
-    {
-      m_link.reset();
-    }
-    m_messageQueue.Clear();
   }
 
+  // Nullable<uint32_t> InitialDeliveryCount;
+
+  // Nullable<uint64_t> MaxMessageSize;
+  // uint32_t MaxLinkCredit{};
```

Comment:
nit: remove commented out code?

Reply:
Those are fields in the options that aren't fully implemented in AMQP layer yet - I plan on cleaning them up later on when I rationalize the uAMQP and Rust AMQP receiver options. But they're not currently necessary.